File: SourceCode/WebSite/API Services/BusinessObjects/WebBusinessInfo.vb
Public MustInherit Class WebBusinessInfo

    ''' <summary>
    ''' Used when you want the business to Redirect The Web To a different page.
    ''' </summary>
    Public Property WebRedirectTo() As String

    ''' <summary>
    ''' Used internally for JSON to Object Serialization.
    ''' </summary>
    Public Property WebObjectType() As String
        Get
            Return Me.[GetType]().ToString()
        End Get
        Set(value As String)
        End Set
    End Property

End Class

File: SourceCode/WebSite/BusinessObjects/WebBusinessInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TotalRecall.BusinessObjects
{
    public abstract class WebBusinessInfo
    {

        /// <summary>
        /// Used when you want to Redirect The Web To a different page.
        /// </summary>
        public string WebRedirectTo { get; set; }

        /// <summary>
        /// Used internally for JSON to Object Serialization.
        /// </summary>
        public string WebObjectType
        {
            get
            {
                return this.GetType().ToString();
            }
            set { }
        }
    }

}